app/scripts/BarTrack.js

```diff
@@ -294,6 +297,78 @@ class BarTrack extends HorizontalLine1DPixiTrack {
     super.zoomed(newXScale, newYScale);
   }
 
+  movedY(dY) {
+    // // see the reasoning behind why the code in
+    // // zoomedY is commented out.
+
+    // Object.values(this.fetchedTiles).forEach((tile) => {
+    //   const vst = this.valueScaleTransform;
+    //   const height = this.dimensions[1];
+
+    //   // clamp at the bottom and top
+    //   if (
+    //     vst.y + dY / vst.k > -(vst.k - 1) * height
```

Comment:
Would it also be okay if we destruct `vst` to `y` and `k`? When you implement `vst` makes sense but when you read this the first time it's a bit hard to follow. I am just thinking of:

```
const { y, k } = this.valueScaleTransform;
```

instead of
```
const vst = this.valueScaleTransform;
```